Makefile.PL

```diff
@@ -486,9 +489,9 @@ sub c_o {
     my $inherited = shift->SUPER::c_o(@_);
 
     if ($^O =~ /Win32/ms && $Config{cc} eq 'cl') {  # Microsoft deprecated -o in favor of /Fo
-        $inherited =~ s{\$\*\.c\n}{/Fo\$\*.obj \$\*.c\n}msg;
+        $inherited =~ s{\$\*\.c[\s\t]*$}{/Fo\$\*.obj \$\*.c\n}msg;
```

Comment:
There is an actual \s at the end of the Makefile line. No way around it. I've changed from [\s\t]* to \s* tho.